src/main/resources/protectedterms/computer_science.terms

```diff
@@ -0,0 +1,5 @@
+JabRef
```

Comment:
This list should be sorted alphabetically. `BibTeX` should be added, too.

Reply:
I fixed the sortation and added `BibTeX` to the list.